src/tooltip.js

```diff
@@ -86,6 +86,10 @@ angular.module('cp-tooltip')
 					tooltipDisplayed = false;
 					if(tooltipEl) tooltipEl.remove();
 				}
+
+				function encodeHTML(str) {
+					return str.replace(/&/g, '&amp;').replace(/</g, '&lt;').replace(/"/g, '&quot;');
+				}
```

Comment:
Of those offered, and looking at what you're doing here, encodeURIComponent is a likely candidate.

> encodeURIComponent("<script>alert('something')</script>")

yields:
"%3Cscript%3Ealert('something')%3C%2Fscript%3E"

But, as @blittle has pointed out already, this is a simplistic approach (which may still be just what you're wanting to do).